eng/common/scripts/modules/Package-Properties.psm1

```diff
@@ -3,11 +3,12 @@
 class PackageProps
 {
     [string]$pkgName
-    [AzureEngSemanticVersion]$pkgVersion
+    [string]$pkgVersion
     [string]$pkgDirectoryPath
```

Comment:
It would definitely be nice if all these properties didn't start with "pkg". It makes using this type a little long winded. You don't have do that update with this PR but it would be nice to update these before they get more usages.

Reply:
Will update this in another PR and also update every location where it is used